_includes/selector.html

```diff
@@ -379,30 +362,26 @@
             active_python_ver: "3.10",
             active_cuda_ver: "11.8",
```

Comment:
Should we default to 12.0 even though it's not available for ARM?
```suggestion
            active_cuda_ver: "12.0",
```

Reply:
I would think we would wait until 12 is supported on all platforms.

Reply:
Agree with Mike, I should have commented that in the code instead of in the comment above.